### ISSUE TITLE
Load Balancer Name Clash

### ISSUE THREAD

**Issue by lukasniemeier-zalando** [zalando-stups/senza#424]
Currently senza generates the load balancer name from stack name and version. In case the name is too long, the stack name component is cut to 32 chars, see https://github.com/zalando-stups/senza/blob/master/senza/components/elastic_load_balancer.py#L18

This can lead to a situation in which senza tries to create two load balancers for the different stacks with the same name.

senza could fallback to generate random name in case the `name-version` combo is already taken.

**Comment by hjacobs** [comment]
@lukasniemeier-zalando there is already a workaround for this by using the `NameSuffix` property on the LB component. Example: https://github.com/zalando-incubator/kubernetes-on-aws/blob/master/cluster/senza-definition.yaml#L99

**Comment by hjacobs** [comment]
I agree that Senza should handle this automatically (e.g. by adding a increasing number if more than one LB was defined).

**Comment by lukasniemeier-zalando** [comment]
Using the `NameSuffix` property solves my issue, thank you.
